rust/core-lib/src/file.rs

```diff
@@ -160,11 +213,74 @@ impl FileManager {
             return Err(FileError::HasChanged(path.to_owned()));
         } else {
             let encoding = self.file_info[&id].encoding;
+            #[cfg(feature = "notify")]
+            self.watcher.unwatch(&path, OPEN_FILE_EVENT_TOKEN);
```

Comment:
This unwatch/watch shouldn't be necessary; this is why we stash the mod time of our own writes, and ignore the corresponding events on load.

Reply:
> ignore the corresponding events on load.
Just to confirm this, we dont want to handle events after I modify and save a tailed file? I tried this in Vscode and it does support the behavior.

Reply:
I'm not totally sure I understand; we want to still receive events after modifying the file, but that's the current behaviour.

Reply:
This is reverted. My test case was wrong.